content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,452 @@
+
+- In _darkroom_, it is now possible to reduce the resolution of the navigation preview, reducing CPU usage and possibly improving performance.
+**_Warning :_** as described in the tooltip of this option, this can slightly reduce the guided mask accuracy.
+
+- In miscellaneous tab, there are new options allow to set shortcuts if multiple instances are used.
+
+- The shortcuts tab now includes a search field that allows you to quickly find the shortcut you want to customize.
+
+And finally, a new lua options tab which, to date, has only one option to show/hide the lua script installer (more on that below).
+
+Note that the modified preferences are saved dynamically. Closing the preferences therefore maintains the modified preferences.
+
+## Lua script manager installer
+
+Bill Ferguson, who deals with the management of Lua scripts, had made a Lua script to manage Lua scripts. Until now, the script manager had to be installed manually.
+
+The author has included, directly in darktable, a Lua installer. This can be enabled or disabled on first boot or in the preferences. The installer downloads the latest available scripts from the darktable Lua scriptsrepository.
+
+Of course, this is only for those who don't already have the manager script installed. This is what this new installer looks like:
+
+  <div style="text-align: center">
+![lua scripts installer]({filename}en/lua-scripts-installer.png)
+  </div>
+
+Once installed, script manager appears on the left panel in lighttable:
+  <div style="text-align: center">
+![script manager]({filename}en/script-manager.png)
+  </div>
+
+## An adjustable histogram and a new mode: RGB parade
+
+Two new features in darktable 3.2 histogram. First is the possibility to adjust histogram height size. This adjustment is done the same way as some lighttable modules: move the cursor over the histogram and press `Ctrl` + `scroll` to adjust the height.
+
+A new mode, RGB parade, appears and displays waveforms that represent the levels of each layer (red, green and blue). This mode allows you to visualize the distribution of color components.
+
+This mode can also be displayed by selecting the histogram display in wave mode (with the top left icon on the histogram) and then  changing the wave mode with the next icon on the right.
+
+  <div style="text-align: center">
+![RGB parade mode]({filename}en/rgb-parade.png)
+  </div>
+
+Add to that an important code rewriting of the histogram for better performances.
+
+## Drawn mask: curved gradient
+
+When adding a drawn mask, it is now possible to bend the gradient shape (gradient). To do this, point the mouse cursor at the gradient bar and scroll up or down with the mouse wheel (or touchpad). `Ctrl` + `scroll` allows you to set the opacity and `Shift` + `scroll` manages the edge hardness.
```

Comment:
Not sure you need the additional "(gradient)". Perhaps it should be earlier in the sentence: "When adding a **gradient** mask..."

Reply:
Really not so important... That will remains as it is.